src/partials/3-boxes/style.scss

```diff
@@ -24,7 +24,7 @@ $yellow-dark: darken($youth-yellow, 15%);
 
   .btn {
     padding: 0.75rem 4rem;
-    background: $youth-yellow;
+    background: $flame-yellow;
     box-shadow: 0 2px 0 $yellow-dark;
```

Comment:
Hadn't noticed, but the `$yellow-dark` variable is silently breaking the stylesheet for `3-boxes`. We can just go for `darken($flame-yellow, 20%)